.github/workflows/documentation.yml

```diff
@@ -0,0 +1,29 @@
+name: Docs
+on: [push, pull_request, workflow_dispatch]
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v3
+      # - uses: Jimver/cuda-toolkit@v0.2.8
+      #   id: cuda-toolkit
+      #   with:
+      #     cuda: '11.7.0'
+      - name: Install dependencies
+        run: |
+          pip install sphinx furo myst_parser sphinx-copybutton
+      - name: Install aestream
+        run: |
+          pip install .
+      - name: Sphinx build
+        run: |
+          sphinx-build -b html docs _build
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        if: ${{ github.event_name == 'push'}}
```

Comment:
I need to double check this,
This should only activate if we push to main and there is some code to do it.
I believe i pasted it in discord at the start of #Docs

Reply:
I changed it to `if: ${{ github.ref == 'refs/heads/main' }}`, which should only push the docs when commits occur on the main branch. I think that's what we want, right?